Neo4j.Driver/Neo4j.Driver/Internal/Result/ResultReaderBuilder.cs

```diff
@@ -23,23 +23,20 @@
 
 namespace Neo4j.Driver.Internal.Result
 {
-    internal class ResultReaderBuilder : IMessageResponseCollector
+    internal class ResultReaderBuilder : BuilderBase
     {
-        private readonly List<string> _keys = new List<string>();
-        private readonly SummaryBuilder _summaryBuilder;
-
         private Func<Task> _receiveOneFunc;
 
         private readonly Queue<IRecord> _records = new Queue<IRecord>();
         private bool _hasMoreRecords = true;
+        public ResultReaderBuilder() : this(null, null, null, null, null)
```

Comment:
Same here. Why we need this constructor? If we will never use this in our code, won't it be better if we have no such a method to protect from using this class wrongly?